File: Python/array_smallest_largest.py
#Find Largest & Smallest Element
arr = list(map(int, input("Enter array elements: ").split()))
max_val = arr[0]
min_val = arr[0]

for num in arr[1:]:
    if num > max_val:
        max_val = num
    if num < min_val:
        min_val = num

print("Largest element =", max_val)
print("Smallest element =", min_val)

File: CPP/array_smallest_largest.cpp
#include<iostream>
using namespace std;
int main()
{
    int arr[5] = { 2,7,4,98,1};
    int max = arr[0];
    int min = arr[0];
    for(int i = 0; i<5; i++){
        if(arr[i]>max)
           max = arr[i];
        if(arr[i]<min)
           min = arr[i];
    }
    cout<<"the largest number : "<<max<<endl;
    cout<<"the smallest number : "<<min<<endl;

    return 0;

}